runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingModeExecutionContext.java

```diff
@@ -591,7 +591,7 @@ public void flushState() {
               timerId,
               "",
               cleanupTime,
-              cleanupTime,
+              window.maxTimestamp(),
```

Comment:
this should be `window.maxTimestamp() + allowedLateness` I think?  cleanupTime here is passed in as `maxTimestamp + allowedLateness + 1ms`

Reply:
@steveniemitz  cleanupTime.minus(1L) added